rst_files/kalman.rst

```diff
@@ -738,8 +738,8 @@ Exercise 3
         # update state and record error
         Ax = A * x
         x = rand(MultivariateNormal(Ax, Q))
-        e1[t] = sum(abs2(a - b) for (a, b) in zip(x, kn.cur_x_hat))
-        e2[t] = sum(abs2(a - b) for (a, b) in zip(x, Ax))
+        e1[t] = sum(abs(a - b)^2 for (a, b) in zip(x, kn.cur_x_hat))
+        e2[t] = sum(abs(a - b)^2 for (a, b) in zip(x, Ax))
```

Comment:
No need for `abs` if you going to `x^2`. `(a-b)^2`

Reply:
That's true as well.